docs/application/tizen-studio/release-notes/3-7-release-notes.md

```diff
@@ -0,0 +1,54 @@
+# Tizen Studio 3.7 Release Notes
+
+- Release Date: Mar 30, 2020
+
+## IDE and Tools
+
+### New Features
+
+- IDE
+  - JDK has been bundled with Tizen Studio removing the dependency of Java being installed in the system.
+  - Support for Chrome 80 has been added.
+  - Log Viewer has been re-enabled in Tizen IDE.
+
+- Certificate Manager
+  - The security of the Tizen certificates has been enhanced by adding support for SHA512 encryption.
+
+### Known Issues
+
+- Tizen Studio
+  - In MacOS, if the UI perspectives are not displayed properly after updating Tizen Studio, it is recommended to restart the IDE. This issue is because of the uncleared cache from previous installation.
+  - In case if you see any old perspective, it is recommended to create a new workspace and import the projects to the new workspace to resolve the issue.
+- Common
+  - If you install Tizen Studio in a directory that requires administrator privileges for access. For example, for C:\ProgramFiles, administrator privileges are required to run the Tizen SDK tools. The Tizen Installer and the baseline SDK Installer alerts you, if you try to install into such a directory.
+- Web and Native IDE
+  - You can create unit tests for Tizen 2.3.2 and higher version projects only. Now Tizen Studio does not support unit testing for older versions.
+- Web IDE
+  - The preview tab in the Web Page Editor sometimes does not appear properly. Use an alternative feature, named Web SDK HTML Editor, which has enhanced features compared to the Web Page Editor. Instead of the preview tab in the Web Page Editor, use the preview feature **Ctrl + 4** of the Web SDK HTML Editor.
+  - In Rapid Development Support (RDS) mode, the web unit test result is not updated.
+- Certificate Manager
+  - Overwriting a duplicate certificate profile in the migration wizard works incorrectly on macOS.
+- Emulator
+  - In the Tizen 5.5 version, the emulator images app un-installation might take some time to complete if the app name contains more than 14 characters. A patch for the same will be released in the next update.
+  - To use Tizen Emulator, use Intel VTx supported CPU and the latest version of the graphic card driver provided by the vendor. To verify the prerequisites for Tizen Emulator, see [Prerequisites for Tizen Studio](../setup/prerequisites.md).
+    - If the host machine is using NVIDIA&reg; Optimus&reg; technology on either Ubuntu or Windows&reg;, you must set the Tizen Emulator to run with your NVIDIA&reg; graphics card. For Ubuntu, verify the [bumblebee project](https://wiki.ubuntu.com/Bumblebee). For Windows&reg;, select **High Speed NVIDIA&reg; Processor** as **Preferred Graphics Processor** in the NVIDIA&reg; control panel.
```

Comment:
I have a question to @journey2w referencing https://github.com/Samsung/tizen-docs/pull/1057

With https://github.com/Samsung/tizen-docs/pull/1057,
"Windows® 10" is updated to "Windows 10".

How about Windows&reg;? Can we keep it or should it be changed to Windows?

Reply:
@adityaaswani 

Could you check the below message ?

I have a question to @journey2w referencing #1057

With #1057,
"Windows® 10" is updated to "Windows 10".

How about Windows®? Can we keep it or should it be changed to Windows?